native_client/deepspeech.h

```diff
@@ -81,7 +81,10 @@ typedef struct Metadata {
   APPLY(DS_ERR_FAIL_CREATE_STREAM,      0x3004, "Error creating the stream.") \
   APPLY(DS_ERR_FAIL_READ_PROTOBUF,      0x3005, "Error reading the proto buffer model file.") \
   APPLY(DS_ERR_FAIL_CREATE_SESS,        0x3006, "Failed to create session.") \
-  APPLY(DS_ERR_FAIL_CREATE_MODEL,       0x3007, "Could not allocate model state.")
+  APPLY(DS_ERR_FAIL_CREATE_MODEL,       0x3007, "Could not allocate model state.") \
+  APPLY(DS_ERR_FAIL_INSERT_HOTWORD,     0x3008, "Could not insert hot-word.") \
```

Comment:
Can you verify if some bindings needs updating of that list? I think at least on .Net side, this might be required.

Reply:
fixed